apstools/beamtime/bss_info_makedb.py

```diff
@@ -0,0 +1,177 @@
+proposal:user3:email	stringout	
+proposal:user3:firstName	stringout	
+proposal:user3:institution	waveform	1024
+proposal:user3:instId	stringout	
+proposal:user3:lastName	stringout	
+proposal:user3:piFlag	bo	
+proposal:user3:userId	stringout	
+proposal:user4:badgeNumber	stringout	
+proposal:user4:email	stringout	
+proposal:user4:firstName	stringout	
+proposal:user4:institution	waveform	1024
+proposal:user4:instId	stringout	
+proposal:user4:lastName	stringout	
+proposal:user4:piFlag	bo	
+proposal:user4:userId	stringout	
+proposal:user5:badgeNumber	stringout	
+proposal:user5:email	stringout	
+proposal:user5:firstName	stringout	
+proposal:user5:institution	waveform	1024
+proposal:user5:instId	stringout	
+proposal:user5:lastName	stringout	
+proposal:user5:piFlag	bo	
+proposal:user5:userId	stringout	
+proposal:user6:badgeNumber	stringout	
+proposal:user6:email	stringout	
+proposal:user6:firstName	stringout	
+proposal:user6:institution	waveform	1024
+proposal:user6:instId	stringout	
+proposal:user6:lastName	stringout	
+proposal:user6:piFlag	bo	
+proposal:user6:userId	stringout	
+proposal:user7:badgeNumber	stringout	
+proposal:user7:email	stringout	
+proposal:user7:firstName	stringout	
+proposal:user7:institution	waveform	1024
+proposal:user7:instId	stringout	
+proposal:user7:lastName	stringout	
+proposal:user7:piFlag	bo	
+proposal:user7:userId	stringout	
+proposal:user8:badgeNumber	stringout	
+proposal:user8:email	stringout	
+proposal:user8:firstName	stringout	
+proposal:user8:institution	waveform	1024
+proposal:user8:instId	stringout	
+proposal:user8:lastName	stringout	
+proposal:user8:piFlag	bo	
+proposal:user8:userId	stringout	
+proposal:user9:badgeNumber	stringout	
+proposal:user9:email	stringout	
+proposal:user9:firstName	stringout	
+proposal:user9:institution	waveform	1024
+proposal:user9:instId	stringout	
+proposal:user9:lastName	stringout	
+proposal:user9:piFlag	bo	
+proposal:user9:userId	stringout	
+""".strip().splitlines()
+
+
+def main():
+    header = """\
+    #
+    # file: bss_info.db
+    # EPICS database for information from APS ESAF & Proposal databases
+    #
+    # BSS: Beamtime Scheduling System
+    #
+    # note: this file autogenerated by make_database.py
+    #    ./bss_info_makedb.py  | tee bss_info.db
+    #
+    # Start the EPICS IOC with a command (from EPICS base):
+    #    softIoc -m P=ioc:bss: -d bss_info.db
+    """
+    for line in header.splitlines():
+        print(line.strip())
+
+    for row in raw_data:
+        parts = row.split()
+        pvname, rtyp = parts[:2]
+        head = f'record({rtyp}, "$(P){pvname}")'
+        fields = []
+        if rtyp == "waveform" and len(parts) == 3:
+            length = parts[-1]
+            fields.append(f'field(FTVL, "CHAR")')
+            fields.append(f'field(NELM, {length})')
+        elif rtyp == "bo":
+            fields.append(f'field(ZNAM, "OFF")')
+            fields.append(f'field(ONAM, "ON")')
+        record = head
+        newline = "\n"
```

Comment:
Codacy found an issue: [local variable 'newline' is assigned to but never used (F841)](https://app.codacy.com/gh/BCDA-APS/apstools/pullRequest?prid=5764884)